IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -54,7 +210,145 @@ - (BOOL)removeItemWithKey:(MSIDTokenCacheKey *)key
                                context:(id<MSIDRequestContext>)context
                                  error:(NSError **)error
 {
-    return nil;
+    
+    assert(key);
+    assert(serializer);
+    
+    MSID_LOG_INFO(context, @"Get keychain items, key info (account: %@ service: %@)", _PII_NULLIFY(key.account), _PII_NULLIFY(key.service));
+    MSID_LOG_INFO_PII(context, @"Get keychain items, key info (account: %@ service: %@)", key.account, key.service);
+    
+    NSMutableDictionary *query = [@{(id)kSecClass : (id)kSecClassGenericPassword} mutableCopy];
+    if (key.service)
```

Comment:
same comment here, maybe this should be a helper method :)

Reply:
added default query